src/renderer/src/Theme.ts

```diff
@@ -86,6 +86,7 @@ const theme = createTheme({
 			},
 		},
 	},
+	spacing: 1,
```

Comment:
a bunch of spacing was being multipied by 8 because of the way the theme api. So setting the spacing to 1 means any padding or margin is multiplied by one. I don't think we have any utility in one defined default spacing

Reply:
Good find! Sorry about that. I probably added that back in the summer and totally forgot why!